api/src/main/resources/workbench-api.yaml

```diff
@@ -0,0 +1,6369 @@
+---
+swagger: '2.0'
+info:
+  version: 0.1.0
+  title: AllOfUs Workbench API
+  description: The API for the AllOfUs workbench.
+  termsOfService: http://www.pmi-ops.org/terms_of_service.html
+  contact:
+    name: developer_help@pmi-ops.org
+  license:
+    name: BSD
+host: api.pmi-ops.org
+securityDefinitions:
+  aou_oauth:
+    authorizationUrl: ''
+    tokenUrl: ''
+    type: oauth2
+    flow: accessCode
+schemes:
+- https
+produces:
+- application/json
+security:
+- aou_oauth: []
+parameters:
+  userId:
+    in: path
+    name: userId
+    type: integer
+    format: int64
+    required: true
+  conceptSetId:
+    in: path
+    name: conceptSetId
+    type: integer
+    format: int64
+    required: true
+    description: Concept set ID
+  dataSetId:
+    in: path
+    name: dataSetId
+    type: integer
+    format: int64
+    required: true
+    description: Data set ID
+  workspaceNamespace:
+    in: path
+    name: workspaceNamespace
+    type: string
+    description: The Workspace namespace
+    required: true
+  workspaceId:
+    in: path
+    name: workspaceId
+    type: string
+    description: The Workspace ID (a.k.a. the workspace's Firecloud name)
+    required: true
+  cohortId:
+    in: path
+    name: cohortId
+    type: integer
+    format: int64
+    required: true
+    description: Cohort ID
+  cohortReviewId:
+    in: path
+    name: cohortReviewId
+    type: integer
+    format: int64
+    required: true
+    description: Cohort Review ID
+  cdrVersionId:
+    in: path
+    name: cdrVersionId
+    type: integer
+    format: int64
+    required: true
+    description: specifies which cdr version
+  participantId:
+    in: path
+    name: participantId
+    type: integer
+    format: int64
+    required: true
+    description: specifies which participant
+paths:
```

Comment:
Dropped a comment that seems important:

```
# Throughout, we use integer/int64 in preference to string/date-time because Swagger's
# date formatting is inconsistent between server and client. Time values are stored as
# milliseconds since the UNIX epoch.
```

Reply:
Let's put this on the top level `info` [object](https://swagger.io/docs/specification/api-general-info/) so we don't lose it again.